arangograph/backups.md

```diff
@@ -38,20 +46,58 @@ It's also possible to create a backup on demand. To do this, click **Back up now
 
 ![Back up Now Dialog](images/arangograph-back-up-now-dialog.png)
 
+If you want to manually copy a backup to a different region than the default
+one, first ensure that the **Upload backup to storage** option is enabled.
+Then, highlight the backup row and use the
+**Copy backup to a different region** button from the **Actions** column. 
+Once the copy of the default backup is created in the region of your choice,
+you can copy it to a different region as well. The source backup ID from
```

Comment:
Ideally, the default backup allows creating multiple copies of itself into multiple regions one at a time if done manually. Maybe this point `Once the copy of the default backup is created in the region of your choice,
you can copy it to a different region as well` can be misleading? What do you think?